package.json

```diff
@@ -17,6 +17,7 @@
         "@changesets/cli": "2.26.1",
         "husky": "8.0.3",
         "is-ci": "3.0.1",
+        "lint-staged": "13.2.1",
```

Comment:
Why add the dependency `lint-staged` to root package.json? For the moment is only being used for `js-client-rest` and can remain there.

Reply:
Thanks a lot! That was my try to run a lint-staged hook. This dependency was a mistake, fixed!